arrow-libs/core/arrow-core/src/commonTest/kotlin/arrow/core/continuations/EffectSpec.kt

```diff
@@ -109,6 +110,32 @@ class EffectSpec :
       }
     }
 
+    "attempt - catch" {
+      checkAll(Arb.int(), Arb.long()) { i, l ->
```

Comment:
Can we add these tests for EagerEffect too? I think we’ll run into the issue that shift cannot be called from the EagerEffect catch method, but I’m not 100% sure.

Reply:
Thank you @serras! I was wrong 😁